packages/grafana-toolkit/src/cli/tasks/plugin.utils.ts

```diff
@@ -69,7 +69,9 @@ const prepareRelease = useSpinner<any>('Preparing release', async ({ dryrun, ver
     ['git', ['config', 'user.email', GIT_EMAIL]],
     ['git', ['config', 'user.name', GIT_USERNAME]],
     await checkoutBranch(`release-${pluginVersion}`),
+    ['cp', ['-rf', distContentDir, 'dist'], { dryrun }],
     ['git', ['add', '--force', distDir], { dryrun }],
+    ['git', ['add', '--force', 'dist'], { dryrun }],
```

Comment:
Why does it add both?
```
/dist/
/ci/dist/
```

Reply:
Just that the example I used when writing this checked in the dist dir. Grafana.com needs a dist dir, so I added both.